biggraphite/drivers/cassandra.py

```diff
@@ -2505,6 +2505,17 @@ def clean(
 
         first_exception = None
 
+        # 0.006% of database
+        num_token_ignore_on_error = 2 ** 50
+
+        env_num_token = environ.get('CLEAN_IGNORE_BATCH_SIZE_ON_ERROR')
```

Comment:
I know that the parameter was also changed, but does it make more sense to have this as a parameter than an environment variable?

Reply:
I decided to get rid of the parameter in favor of env. variable because those ones are cassandra-only related; Adding a parameter requires to also change function definitions in hybrid, memory, elasticsearch, and there is no real point of them.  Also, as I really want to fine tweak cassandra cleaning, I might add new configurable settings the same way (and this would make functions & parameter passing really huge).
I'll only need to document those (why default value).